src/serve/javaEmulators.ts

```diff
@@ -88,13 +115,24 @@ function _getCommand(emulator: JavaEmulators, args: { [s: string]: any }): JavaE
     const argKey = "--" + key;
     const argVal = args[key];
 
-    cmdLineArgs.push(argKey, argVal);
+    if (!argVal) {
```

Comment:
how likely is this to misbehave if I pass in a boolean like `--fork=false`?

Reply:
Very ... good call.